client/templates/index.html

```diff
@@ -1,5 +1,6 @@
 <head>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script src="//cdn.ckeditor.com/4.6.2/standard/ckeditor.js"></script>
```

Comment:
As explained on slack:
A relative URL without a scheme (http: or https:) is valid, per RFC 3986: "Uniform Resource Identifier (URI): Generic Syntax", Section 4.2.